resources/js/login/login.js

```diff
@@ -114,9 +117,46 @@ const PasswordInputForm = ({
                     )
                 }}
             />
-            {passwordError &&
-                <p className={styles.error_label} dangerouslySetInnerHTML={{__html: passwordError}}></p>
-            }
+            {(() => {
+                const attempts = parseInt(loginAttempts, 10);
```

Comment:
https://github.com/OpenStackweb/openstackid/pull/82/files#r2204813659
Would be great to get this number from one single point in server